core/templates/dev/head/player/exploration_player.html

```diff
@@ -95,7 +96,8 @@ <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
               src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
             <li ng-repeat="ContributorUsername in explorationContributorUsernames| limitTo: 2"
               tooltip="<[ContributorUsername]>" tooltip-placement="top">
-              <img class="oppia-contributors-circle" ng-src="/images/avatar/user_mint_48px.png" alt="">
+              <profile-link-image username="<[ContributorUsername]>" classes="oppia-contributors-circle" alt="">
```

Comment:
Done re:alt.

As for the class question, in terms of sizing it does make a difference for this particular instance-- the circle doesn't quite line up with the rest of the icons without the oppia-contributors-circle class. I agree self-contained is good, but do we want to blend functional components and style? Like the more customization you can put on top of the link the more reusable it is, right? Those are my initial thoughts, could totally be off.